packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -746,7 +746,7 @@ abstract class DatabaseInstanceNew extends DatabaseInstanceBase implements IData
       performanceInsightsRetentionPeriod: enablePerformanceInsights
         ? (props.performanceInsightRetention || PerformanceInsightRetention.DEFAULT)
         : undefined,
-      port: props.port?.toString(),
+      port: props.port ? Tokenization.stringifyNumber(props.port) : undefined,
```

Comment:
Two things:

1. I think you want the condition to be `props.port === undefined`, as 0 is falsy in JavaScript.
2. I believe you can use `Token.toString(props.port)` instead - it's more of the idiomatic way to do it (`Tokenization` is more of an internal class).

Reply:
1. Good catch. 0 is a valid port number. I changed it.
2. Method `Token.toString()` doesn't accept an attribute. So I can't use it. Should I try something else? Or is `Tokenization.stringifyNumber(props.port)` the only way to handle it?